pygfunction/gfunction.py

```diff
@@ -1531,238 +1538,3 @@ def uniform_temperature(boreholes, time, alpha, nSegments=8,
         boundary_condition=boundary_condition, options=options)
 
     return gFunc.gFunc
-
-
-def equal_inlet_temperature(
-        boreholes, UTubes, m_flow_borehole, cp_f, time, alpha,
-        kind='linear', nSegments=8, segment_ratios=segment_ratios,
-        use_similarities=True, disTol=0.01, tol=1.0e-6, dtype=np.double,
-        disp=False, **kwargs):
```

Comment:
These are all tagged for deprecation. I removed them so I didn't have to continue to deal with circular import problems.